Sources/Evangelium/Helpers/ReadingFactory.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  ReadingFactory.swift
+//  Evangelium
+//
+//  Created by Manuel Sánchez on 2/4/20.
+//
+
+import Foundation
+
+struct ReadingFactory {
+    static func create(from todayReadings: [Reading], with date: String) -> SwiftyReading{
+        let readings = todayReadings.filter { reading in reading.type == "reading" }
```

Comment:
for the `type` it would be better if you use enum for represent the different types you could have